scripts/generate_driver_wrappers.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""This script is required for the auto generation of the
+   psa_crypto_driver_wrappers.c file"""
```

Comment:
This will be called by build scripts. Typically from the makefile or cmakefile that we provide, but some users have their own build scripts.

Reply:
For users who have their own buildscripts, this will need to be integrated just the way the other autogen files are like error.c  etc ?
Do we provide some specific guidance for such users ?